db/migrate/20230415011209_create_ta_mappings.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTaMappings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :ta_mappings do |t|
+      t.references :course, null: false, foreign_key: true
+      t.integer :ta_id
```

Comment:
Please avoid directly creating a column for referencing a foreign key instead use reference.

In my opinion, it would be less confusing if we used `user_id` instead of `ta_id` in the TaMappings table. This would better reflect the ID of the TaMapping entry.

Reply:
sure, will change the name